_potd_111_enemies/slime.md

```diff
@@ -13,12 +17,12 @@ vulnerabilities:
   stun: true
   resolution: false
 abilities:
-  - name: Acid Spray
-    description: 'applies stacking physical vulnerability up; this is used
-    instead of auto-attacks, but does magical damage, so does not affect its
-    own damage'
   - name: 'Rapture'
-    description: 'Enrage; explodes, doing huge AoE damage; used 37 seconds after pull/agro'
+    potency: 100% of max HP
+    description: 'instant AoE sacrificial enrage'
```

Comment:
Fixed - I auto-filled a lot of the Deep Palace enemies abilities from the Palace versions and probably overlooked this when merging the ability lists (rather silly since I pointed it out in the first place, lol). Thanks for catching this!